### PR TITLE
fix typo in balloon_stress test case

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: qemu/tests/balloon_stress.py [hunk 1]
```diff
@@ -75,11 +75,11 @@ def run(test, params, env):
         for memory in xrange(start, end, step):
             logging.debug("ballon vm mem to: %s B" % memory)
```
Comment (ypu) [inline qemu/tests/balloon_stress.py]: Seems still have some typo left in the logging part like this line.

**Comment by luckyh** [comment]
looks good, acked.

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by spcui** [comment]
LGTM
Acked-by: @spcui

**Comment by ypu** [comment]
Thanks @luckyh @CongSmile and @spcui will merge it.

**Pull request merged**